packages/spotlight/src/spotlight.js

```diff
@@ -354,6 +354,12 @@ const Spotlight = (function () {
 		_pointerMoveDuringKeyPress = false;
 	}
 
+	function webOSMouseHandler (ev) {
```

Comment:
What do you think about renaming this `onWebOSMouse`?

Reply:
Handlers should be `handleXxxx` not `onXxxxx`.

Reply:
it's handleWebOSMouseEvent now.